js/main.js

```diff
@@ -22,6 +22,15 @@ const { deleteIndexFolder } = require('./search/search.js');
 
 require('electron-dl')();
 
+// ELECTRON-261: On Windows, due to gpu issues, we need to disable gpu
+// to ensure screen sharing works effectively with multiple monitors
+// https://github.com/electron/electron/issues/4380
+if (isWindowsOS) {
+    app.commandLine.appendSwitch("disable-gpu", true);
```

Comment:
so we are doing this always???  isn't the problem only related to screensharing?  is there a way to only disable when sharing a screen?  what is the perf impact to user that have gpu - will the electron wrapper perform more poorly. 

also does this happen in all versions of os - or just win10 - why not then just do for win10?

maybe another option is selectively allow users to disable?  maybe a user feature or an install option???

Reply:
1. This cannot be specifically turned on or off for a screen sharing session because the flags only get enabled before the "on-ready" event from the main process.

2. I'll profile the app and get back with the performance metrics.

3. Makes sense to limit this to Windows 10. Will make necessary changes.

4. Great idea to have this selectively enabled by the administrators from the config file. Will check with product to take their input.

Thanks for the feedback.